Cargo.toml

```diff
@@ -41,8 +44,8 @@ path = "src/main.rs"
 [dependencies]
 anyhow = "=1.0.86"
 guppy = "=0.17.5"
-log = "=0.4.21"
-loggerv = "=0.7.2"
+itertools = "0.12.1"
```

Comment:
This line looks like a merge conflict, the itertools dependency was removed a few days ago:

```suggestion
```

This is not blocking, if you don't get to it we can just remove the dependency after this PR merges